lib/parse.js

```diff
@@ -44,9 +44,9 @@
 
 "use strict";
 
-var KEYWORDS = 'break case catch class const continue debugger default delete do else export extends finally for function if in instanceof new return switch throw try typeof var let void while with import';
+var KEYWORDS = 'break case catch class const continue debugger default delete do else export extends finally for function if in instanceof new return switch throw try typeof var let void while with';
 var KEYWORDS_ATOM = 'false null true';
-var RESERVED_WORDS = 'enum implements interface package private protected public static super this ' + KEYWORDS_ATOM + " " + KEYWORDS;
+var RESERVED_WORDS = 'enum implements import interface package private protected public static super this' + " " + KEYWORDS_ATOM + " " + KEYWORDS;
```

Comment:
Why `' + " "` instead of ` '`? :sweat_smile:

Reply:
I'll fix that.